docs/validate/setup-and-configuration/state-sync.md

```diff
@@ -234,16 +325,40 @@ sed -i -e "s%^persistent_peers *=.*%persistent_peers = \"$PEER\"%; " $HOME/.evmo
 
 Set the variables for start from snapshot
 
+:::tip
+**Note**: Usually, on other cosmos chains, the user is instructed to use as trusted height
+the latest height minus 2000 by default.
+However, snapshots of the Evmos chain take a long time to generate, which makes the gap between the snapshot height and
+the current latest height be more than 2000.
+This results in a context timeout error:
+
+```
+5:33PM ERR error on light block request from witness, removing... error="post failed: Post \"http://bd-evmos-mainnet-state-sync-us-01.bdnodes.net:26657\": context deadline exceeded" module=light primary={} server=node
+```
+
+To avoid this issue, simply pick a trusted height close to snapshot height.
+For example, if you know there's a snapshot at height `13286000`, consider a trust hash from block `13284000`.
+
+You can get the latest snapshot height from [Polkachu here.](https://polkachu.com/tendermint_snapshots/evmos).
+Otherwise, your node will find the available snapshots.
+You will see logs similar to this:
+
+```
+5:48PM INF Discovered new snapshot format=2 hash="�z���Մ��^��Q\x1a\\I_�\x0f�OT!�(jM�$!��" height=13286000 module=statesync server=node
+```
+
+:::
+
 ```bash
-LATEST_HEIGHT=$(curl -s $SNAP_RPC/block | jq -r .result.block.header.height); \
-BLOCK_HEIGHT=$((LATEST_HEIGHT - 2000)); \
+LATEST_SNAPSHOT_HEIGHT=<fill_with_the_latest_snapshot_height>; \
```

Comment:
yeah, we can leave that as it is right now. I think the key point here is to communicate that the trusted height should be close to the latest snapshot height. Otherwise the state sync will throw a timeout error